Tests/Data/Parabolic/T/3D_2U_BHE/3D_2U_BHE.prj

```diff
@@ -63,6 +54,94 @@
             </borehole_heat_exchangers>
         </process>
     </processes>
+    <media>
+        <medium id="0">
+            <phases>
+                <phase>
+                    <type>AqueousLiquid</type>
+                    <properties>
+                        <property>
+                            <name>specific_heat_capacity</name>
+                            <type>Constant</type>
+                            <value>4068</value>
+                        </property>
+                        <property>
+                            <name>thermal_conductivity</name>
+                            <type>Constant</type>
+                            <value>0.1</value>
+                        </property>
+                        <property>
+                            <name>density</name>
+                            <type>Constant</type>
+                            <value>992.92</value>
+                        </property>
+                    </properties>
+                </phase>
+                <phase>
+                    <type>Solid</type>
+                    <properties>
+                        <!-- <property> -->
+                            <!-- <name>permeability</name> -->
+                            <!-- <type>Constant</type> -->
+                            <!-- <value>1.e-14 0 0 1.e-14</value> -->
+                        <!-- </property> -->
+                        <!-- <property> -->
+                            <!-- <name>porosity</name> -->
+                            <!-- <type>Constant</type> -->
+                            <!-- <value>0.001</value> -->
+                        <!-- </property> -->
```

Comment:
These properties seems to be unused currently. I strongly recommend to drop the unused, commented code parts.

Reply:
Same as the above. Removed for now.